ui/components/ui/icon-button/icon-button.stories.tsx

```diff
@@ -34,4 +34,15 @@ type Story = StoryObj<typeof IconButton>;
 
 export const Default: Story = {};
 
-Default.storyName = 'Default';
+export const Disabled: Story = {
+  args: {
+    disabled: true,
+  },
+};
+
+export const LongLabel: Story = {
+  args: {
+    label: 'This is a very long label that should trigger the tooltip',
+  },
+};
+
```

Comment:
Adding a couple more stories to `IconButton` in storybook to check different states

<img width="654" alt="Screenshot 2025-04-24 at 3 54 04 PM" src="https://github.com/user-attachments/assets/3c94d19d-35dd-4ab7-a64c-525178c738da" />
<img width="479" alt="Screenshot 2025-04-24 at 3 54 00 PM" src="https://github.com/user-attachments/assets/2729c57e-0be5-4c11-b099-b48b27219d95" />